.flake8

```diff
@@ -0,0 +1,33 @@
+[flake8]
```

Comment:
This is in develop already, wtf GitHub?  Here because I rebased *this* on develop after the .flake8 PR merge

Reply:
I had this annoyance too. Fix is `rebase -i` and removing  the commit in the rebase (or it was for me)